src/app/home/home.component.scss

```diff
@@ -0,0 +1,12 @@
+.flex-container {
```

Comment:
mozna to ogarniac fajnie klasami ktore dostarcza bootstrap ( d-flex justify-content-center)

Reply:
sluszna uwaga, zrobione